packages/oui-field/src/field.html

```diff
@@ -0,0 +1,31 @@
+<div class="oui-field">
+    <label class="oui-field__label oui-label"
+        ng-attr-for="{{ ::$ctrl.for }}"
+        ng-if="$ctrl.label"
```

Comment:
one time binding could be appropriate

Reply:
One time binding has no effet with ng-if